inference-engine/src/inference_engine/include/ie/ie_plugin_config.hpp

```diff
@@ -283,6 +283,16 @@ DECLARE_CONFIG_VALUE(CPU_THROUGHPUT_AUTO);
  */
 DECLARE_CONFIG_KEY(PERF_COUNT);
 
+/**
+ * @brief The name for setting work mode in MULTI device plugin option.
+ *
+ * It is passed to Core::SetConfig(), this option should be used with value only:
+ * PluginConfigParams::AUTO or PluginConfigParams::MULTI
+ */
+DECLARE_CONFIG_KEY(WORK_MODE);
+DECLARE_CONFIG_VALUE(AUTO);   // fixme: correct me with more meaningful name, thanks!
```

Comment:
on the names: maybe need to better indicate that these are MULTI specific, e.g. MULTI_MODE
with MULTI_MODE_AUTO and MULTI_MODE_LEGACY?

Reply:
ok, will change the value name with MULTI_MODE_AUTO and MULTI_MODE_LEGACY